### PR TITLE
USHIFT-1459: Add periodic job for metal tests

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-ocp-metal-nightly

**Comment by pacevedom** [comment]
/retitle USHIFT-1459: Add periodic job for metal tests

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-ocp-metal-nightly

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-ocp-metal-nightly

File: ci-operator/jobs/openshift/microshift/openshift-microshift-main-periodics.yaml [hunk 1]
```diff
@@ -13,14 +86,6 @@ periodics:
...
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-microshift-main-rebase-on-nightlies
-  reporter_config:
-    slack:
-      channel: '#microshift-alerts'
-      job_states_to_report:
-      - failure
-      - error
-      report_template: '<@here> :red_jenkins_circle: Job *{{.Spec.Job}}* ended with
-        *{{.Status.State}}*. <{{.Status.URL}}|View logs>'
```
Comment (pmtk) [inline ci-operator/jobs/openshift/microshift/openshift-microshift-main-periodics.yaml:23]: Do we want to remove that?

Comment (pacevedom) [reply ci-operator/jobs/openshift/microshift/openshift-microshift-main-periodics.yaml:23]: well, this came out of running make jobs. Havent seen changes to the rebase so it must come from somewhere else. In a different PR I will set up slack alerts so I can add them back.

Comment (pmtk) [reply ci-operator/jobs/openshift/microshift/openshift-microshift-main-periodics.yaml:23]: https://docs.ci.openshift.org/docs/how-tos/notification/ doesn't explicitly say that, but I think it needs to be added manually to the job file.
[ci-operator's config](https://steps.ci.openshift.org/ci-operator-reference) does not include any `reporter`

These alerts were added by @eggfoobar in [this PR](https://github.com/openshift/release/pull/38115) - Egli could you chime in?

Comment (pacevedom) [reply ci-operator/jobs/openshift/microshift/openshift-microshift-main-periodics.yaml:23]: I was just reading that too, let me add it back.

Comment (pacevedom) [reply ci-operator/jobs/openshift/microshift/openshift-microshift-main-periodics.yaml]: Done.

Comment (eggfoobar) [reply ci-operator/jobs/openshift/microshift/openshift-microshift-main-periodics.yaml]: Yup! To my knowledge those still need to be added manually, thanks @pacevedom !

**Pull request merged**